cdap-app-fabric/src/main/java/co/cask/cdap/app/DefaultApplicationContext.java

```diff
@@ -26,8 +30,25 @@
  */
 public class DefaultApplicationContext<T extends Config> implements ApplicationContext<T> {
 
+  private static final Gson GSON = new Gson();
+  private final T appConfig;
+
+  public DefaultApplicationContext() {
+    this.appConfig = null;
+  }
+
+  //TODO : Handle JsonSyntax Exception better and throw a more meaningful error. This will be done when we
```

Comment:
is there a jira open for this? if so lets put it in the comment, otherwise lets open one and then put it in the comment.